File: core/src/main/java/forpleuvoir/dhwu1a/core/event/message/OtherClientMessageEvent.java
package forpleuvoir.dhwu1a.core.event.message;

import forpleuvoir.dhwu1a.core.message.OtherClientMessage;
import forpleuvoir.dhwu1a.core.user.base.User;

/**
 * 其他客户端消息事件
 *
 * @author forpleuvoir
 * <p>#project_name dhwu1a
 * <p>#package forpleuvoir.dhwu1a.core.event.message
 * <p>#class_name OtherClientMessageEvent
 * <p>#create_time 2021/7/4 12:05
 */
public class OtherClientMessageEvent extends MessageEvent<OtherClientMessage> {

    protected OtherClientMessageEvent(OtherClientMessage message) {
        super(message);
    }

    @Override
    public User getUser() {
        return null;
    }
}

File: core/src/main/kotlin/forpleuvoir/dhwu1a/core/event/message/OtherClientMessageEvent.kt
package forpleuvoir.dhwu1a.core.event.message

import forpleuvoir.dhwu1a.core.message.OtherClientMessage
import forpleuvoir.dhwu1a.core.user.base.User

/**
 * 其他客户端消息事件
 *
 * @author forpleuvoir
 *
 * #project_name dhwu1a
 *
 * #package forpleuvoir.dhwu1a.core.event.message
 *
 * #class_name OtherClientMessageEvent
 *
 * #create_time 2021/7/4 12:05
 */
class OtherClientMessageEvent(message: OtherClientMessage) : MessageEvent<OtherClientMessage>(message) {

    override val user: User?
        get() = null
}